ClinGen/clincoded
# Connects to #1626. Connects to #1628. Connects to #1629.

**Steps to test #1626:**
1. Login as part of an affiliation, and curate 3 proband evidence with 3 different variants in a GDM.
2. At the top of the gene-disease record page, click on the individual variant link buttons to curate each variant.
3. Upon curating the variants, click on each of the variant link buttons to go back into each of the curated variants. Expect to see each of the different variants retaining their own curation info.
4. Return to the Dashboard page and change to a different affiliation.
5. Go to the same GDM and curate the same 3 variants.
6. Expect to see each of the different variants retaining their own curation info, as well as showing the correct curations from the prior affiliation.

**Steps to test #1628:**
1. In a GDM, curate and score a proband evidence.
2. Then edit the scored proband evidence by removing the score.
3. Click the **View/Score** link for the proband evidence in the Curation Palette.
4. Expect to see the view-only evidence page to be rendered properly.
5. Removing the score can also be tested from the view-only page.
6. Repeat the testing steps for Case-Control and Experimental evidence and expect to see the same behaviors.

**Steps to test #1629:**
1. Curate and score 3 proband evidence in a GDM with various gene impact types and points involving 0.1 scores.
2. Click on the blue Classification Matrix button at the upper right of the gene-disease record page.
3. In the Classification Matrix table, expect to see the evidence total points and points counted to be rounded to 1 or 2 decimals (e.g. **0.3** or **0.45**) rather than **0.300000000000004**.
